stock_prices/stock_prices.py

```diff
@@ -2,14 +2,29 @@
 
 import argparse
 
+
 def find_max_profit(prices):
-  pass
+    profit = 0
+
+    for i in range(len(prices) - 1):
+        for j in range(i + 1, len(prices)):
+            if prices[j] - prices[i] > profit:
+                profit = prices[j] - prices[i]
+
+    return profit
```

Comment:
I also had a double-for loop for my solution. However, you need to account for the negative answer in the test file when all of the "profits" are negative.